README.md

```diff
@@ -244,6 +244,20 @@ import {formatFileSize} from '@brightspace-ui/intl/lib/fileSize.js';
 const fileSize = formatFileSize(100); // -> '100 bytes' in en-US
 ```
 
+## List Formatting
+
+Use `getSeparator` to get the appropriate list separator for the current locale. This is a separator that would be used in spoken language; note that the separator includes a space, for locales where it is appropriate.
+
+```javascript
+import {getSeparator} from '@brightspace-ui/intl/lib/list.js';
```

Comment:
Just because our current lint rules will complain about this if people copy-paste:

```suggestion
import { getSeparator } from '@brightspace-ui/intl/lib/list.js';
```

Reply:
Okay - I followed the formatting in the rest of the readme, should I update the spacing for all the imports?

Also, I think our readme is a bit confusing in its treatment of `options`, which is made to seem like an array - e.g.:

```js
const number = formatNumber(8902.72, [options]);
```

Reply:
> should I update the spacing for all the imports?

Only if you want. Likely our linting rules have gotten a bit stricter since this was originally written.

> I think our readme is a bit confusing in its treatment of options

The square brackets were meant to mean "optional parameter", but since they also mean an array that's confusing for sure.